app/src/main/java/org/p2p/wallet/newsend/NewSendPresenter.kt

```diff
@@ -283,21 +285,30 @@ class NewSendPresenter(
     }
 
     override fun send() {
+        val feeState = feeRelayerManager.getState()
+        if (feeState !is FeeRelayerState.UpdateFee) return
```

Comment:
Weird that send depends on state.
We should move this to manager or calculation mode and show loading buy callback